### ISSUE TITLE
[A11Y] Android:Images on the components page are unlabeled

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#387]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet):  Nexus 9
Browser: Chrome Dev 59.0.3062.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to COMPONENTS link and double tap on it.
3. Try to navigate through the images on the page.

**ACTUAL RESULTS:**

Images are unlabeled and talkback reads the filename of the images.

**EXPECTED RESULTS:**

Images should have appropriate labels. On this page, captions used for images can be tagged as labels and screen reader focus needs to be disabled for the displayed caption text to avoid redundant reading of caption text.

**ATTACHMENT(S):**

<img width="1440" alt="unlabeled_img1" src="https://cloud.githubusercontent.com/assets/26986019/24921182/c9bdaa28-1e9e-11e7-963a-a624e7d4bf49.png">
<img width="1440" alt="unlabeled_img2" src="https://cloud.githubusercontent.com/assets/26986019/24921183/c9be49b0-1e9e-11e7-9047-2de5988489ec.png">
<img width="1440" alt="unlabeled_img3" src="https://cloud.githubusercontent.com/assets/26986019/24921184/c9c00476-1e9e-11e7-9b0d-75b725cd8acf.png">

**Comment by torch2424** [comment]
Would like #492 to be pulled before doing this to avoid large merge conflicts.
